include/mbedtls/ssl_internal.h

```diff
@@ -165,8 +183,12 @@ struct mbedtls_ssl_handshake_params
     /*
      * Handshake specific crypto variables
      */
-    int sig_alg;                        /*!<  Hash algorithm for signature   */
-    int verify_sig_alg;                 /*!<  Signature algorithm for verify */
+
+#if defined(MBEDTLS_SSL_PROTO_TLS1_2) && \
+    defined(MBEDTLS_KEY_EXCHANGE__WITH_CERT__ENABLED)
+    mbedtls_ssl_sig_hash_set hash_algs;             /*!<  Set of suitable sig-hash pairs */
+#endif
+    int verify_sig_alg;                         /*!<  Signature algorithm for verify */
```

Comment:
That's a takeover from previous versions, I haven't yet checked the use of this member.